### ISSUE TITLE
Finish or discard useComputed ?

### ISSUE THREAD

**Issue by FredyC** [mobxjs/mobx-react-lite#4]
Continuing from [my monolog ](https://github.com/mobxjs/mobx-react-lite/issues/1#issuecomment-437624590). The implementation (and test) are in the repo, but not quite working. Would love to hear some opinions if it's needed and what's wrong with it or ditch it completely.

**Comment by mweststrate** [comment]
I think `useComputed` could be written as follows, using a boxed computed value and maintaining it's reference:

```javascript
export function useComputed<T>(initialValue: () => T): T {
    const computed = useMemo(() => computed(initialValue), [])
    return computed.get()
}
```

**Comment by FredyC** [comment]
I knew there must be some easier approach :) It's seemingly working indeed. However, there is a small caveat. If the computation is based on props values (or anything non-observable), it won't work as the `useMemo` won't be refreshed.

That means we have to expose `inputs` for `useMemo` as well and it's not that pretty anymore. What do you think?

**Comment by mweststrate** [comment]
Good point, yeah if it depends on props or other pieces of non mobx state then we have to expose inputs. But at least we can stay away from having to make props / state observable like in the classic bindings :)

**Comment by FredyC** [comment]
Fixed `useComputed` released in v0.2.0
